csrc/quantization/aqlm/aqlm_cuda_entry.cpp

```diff
@@ -0,0 +1,166 @@
+// Adapted from https://github.com/Vahe1994/AQLM
+
```

Comment:
A question I have here is do we need to include a more official license here?  Something including NM as well?

Reply:
@robertgshaw2-neuralmagic ?

Reply:
Hmm, how would that be done?  Nevermind, I'll take a look at the Marlin PR.

Reply:
So how's this?

Reply:
@mgoin and @robertgshaw2-neuralmagic

Reply:
great!